src/ruby/ext/symengine/extconf.rb

```diff
@@ -0,0 +1,74 @@
+require 'mkmf'
```

Comment:
Shall I try without the extconf.rb then? I am not sure if I can do it without extconf.rb, because the tutorials and docs that I follow use extconf.rb. I haven't learnt using cmake yet.